### PR TITLE
[prepare-content, validate] - add support for supportlevelheader

### PR DESCRIPTION
**Pull request by GuyAfik**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-8452

## Description
see issue's description

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Modified the `validate` and `format` pre-commit hooks to run with the `--all` flag only when the `--mode=nightly` argument and `--all` flag were given.
 * Modified the `ruff` pre-commit hook to run with `--config=nightly_ruff.toml` argument when running **pre-commit** command wite the `--mode=nightly` argument.
 * Fixed an issue where deprecating parsing rules or modeling rules using **format** failed due to schema discrepancies.
+* Updated the **prepare-content** to add a clarification to the `detaileddescription` field of partner event collectors that they are under the support of PANW.
```
Comment (ShirleyDenkberg) [inline CHANGELOG.md:7]:
```suggestion
* Updated the ***prepare-content*** command to add a clarification to the `detaileddescription` field of partner event collectors that they are under the support of PANW.
```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz Doc review completed.

File: demisto_sdk/commands/common/schemas/integration.yml [hunk 2]
```diff
@@ -56,6 +56,9 @@ mapping:
     type: str
   autoconfiginstance:
     type: bool
+  supportlevelheader:
+    type: str
+    enum: ['xsoar', 'partner', 'community']
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/schemas/integration.yml:61]: will this break if the header isnt present?

**Review by JudahSchwartz — APPROVED** [review]

File: demisto_sdk/commands/common/schemas/integration.yml (hunk 2 above)
Comment (GuyAfik) [reply demisto_sdk/commands/common/schemas/integration.yml:61]: no, because the **supportlevelheader** is **optional**.

**Pull request merged**